test/export/report.test.ts

```diff
@@ -170,12 +170,14 @@ describe("report", () => {
             });
 
             it("should return error when polling fail", () => {
+                jest.setTimeout(5000);
```

Comment:
https://jestjs.io/docs/en/timer-mocks fake timers or something else https://github.com/gooddata/gooddata-js/blob/master/test/report.test.ts#L220

Reply:
It works now since we refactor to `async/await`
Still a magic to me :(